OWNERS_ALIASES

```diff
@@ -9,9 +9,6 @@ aliases:
     - tisutisu
     - tnevrlka
     - MartinBasti
```

Comment:
@MartinBasti do you want to be moved just to prefetch as well?

Reply:
proabably yes, eventually I'll be thrown there next year

Reply:
I meant to keep me :D (is Friday already?)